console/console-init/ui/src/modules/iot-project-detail/Routes.tsx

```diff
@@ -5,18 +5,31 @@
 
 import React from "react";
 import { SwitchWith404, LazyRoute } from "use-patternfly";
-import { Redirect } from "react-router";
 
 const getCertificates = () =>
   import("modules/iot-certificates/IoTCertificatesPage");
 const getDevices = () => import("modules/iot-device/DeviceListPage");
+const getProjectDetailPage = () =>
```

Comment:
I will suggest name as getIoTProjectDetailPage as it will create confusion in messaging and iot projects